packages/core/aqua-to-js/src/utils.ts

```diff
@@ -27,24 +27,25 @@ import {
   SimpleTypes,
   UnlabeledProductType,
 } from "@fluencelabs/interfaces";
+import { z } from "zod";
 
-export interface PackageJson {
-  name: string;
-  version: string;
-  devDependencies: {
-    ["@fluencelabs/aqua-api"]: string;
-  };
-}
+const packageJsonSchema = z.object({
+  name: z.string(),
+  version: z.string(),
+  devDependencies: z.object({
+    ["@fluencelabs/aqua-api"]: z.string(),
+  }),
+});
+
+export type PackageJson = z.infer<typeof packageJsonSchema>;
 
 export async function getPackageJsonContent(): Promise<PackageJson> {
   const content = await readFile(
-    new URL(path.join("..", "package.json"), import.meta.url),
+    new URL(join("..", "package.json"), import.meta.url),
```

Comment:
some doc on why you use `../package.json` path, and why you believe it will be there

Reply:
I see you liked, but I couldn't find any doc folex suggested adding